public/components/visualize/wz-visualize.js

```diff
@@ -275,11 +256,8 @@ export const WzVisualize = compose (withErrorBoundary,withReduxProvider) (class
     return (
       <Fragment>
         {/* Sample alerts Callout */}
-        {this.state.thereAreSampleAlerts && this.props.resultState === 'ready' && (
-          <EuiCallOut title='This dashboard contains sample data' color='warning' iconType='alert' style={{ margin: '0 8px 16px 8px' }}>
-            <p>The data displayed may contain sample alerts. Go <EuiLink href='#/settings?tab=sample_data' aria-label='go to configure sample data'>here</EuiLink> to configure the sample data.
-            </p>
-          </EuiCallOut>
+        {this.props.resultState === 'ready' && (
+          <SampleData context={`${WzVisualize.name}-sample-data`} />
```

Comment:
Maybe this context is not necessary if you only use it for the orchestrator, because the context must be taken in the place where the error is caught, as I mentioned before. 🤓

Reply:
Removed it